websites/docs-smoke-test/src/data/navigation.yaml

```diff
@@ -19,6 +19,8 @@
       path: /components/images
     - title: Code Examples
       path: /components/code-examples
+    - title: Custom Anchor
+      path: /components/custom-anchor
```

Comment:
Nit: probably better to put those in the "Views" chapter, as they relate more to demoing the use cases (see links as an example)